src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -16,7 +16,7 @@
 public class StorageFile {
 
     /** Default file path used if the user doesn't provide the file name. */
-    public static final String DEFAULT_STORAGE_FILEPATH = "addressbook.txt";
+    public static final String DEFAULT_STORAGE_FILEPATH = "addressbook";
```

Comment:
what's the rationale of removing the extension? since in main.java you give the default value of 'addressbook.txt' as well.